src/core/sys/posix/libgen.d

```diff
@@ -0,0 +1,20 @@
+/**
+ * D header file for POSIX.
+ *
+ * Copyright: 2016 Sociomantic Labs GmbH
+ * License:   <a href="http://www.boost.org/LICENSE_1_0.txt">Boost License 1.0</a>.
+ * Authors:   Leandro Lucarella
+ * Standards: The Open Group Base Specifications Issue 6, IEEE Std 1003.1, 2004 Edition
+ */
+
+/*          Copyright Sociomantic Labs GmbH 2016.
+ * Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
+ */
+module ocean.stdc.posix.libgen;
+
+extern (C):
+
+char* basename(char*);
+char* dirname(char*);
```

Comment:
Noted, but that's not the style of most C headers in druntime, so I prefer to stick with the common practices in the project :)